tests/benchmarkstt/test_metrics_core.py

```diff
@@ -24,16 +24,18 @@ def test_diffcounts(a, b, exp):
 
 @pytest.mark.parametrize('a,b,exp', [
     # (strict_wer, hunt_wer)
```

Comment:
It would be good to update this comment with the variables that are being used in this test
```suggestion
    # (wer_strict, wer_hunt, wer_levenshtein)
```

Reply:
Absolutely!